index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+    ul {
+      list-style-type: none;
+      margin: 0;
+      padding: 0;
+    }
+    p{
+      font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+      font-size: large;
+      margin-left: 15%;
+      margin-right: 15%;
+      padding-top: 15%;
+    }
+    #welcome-section{
+      height: 100vh;
+    }
+    #navbar {
+  overflow: hidden;
+  background-color: lightyellow;
+  position: fixed; /* Set the navbar to fixed position */
+  top: 0; /* Position the navbar at the top of the page */
+  width: 100%; /* Full width */
+    }
+@media only screen and (max-width: 4180px) {
+  body {
+    background-color: lightblue;
+  }
+}
+      
+    </style>
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#about">About</a></li>
+        <a href ="https://github.com/shahzaaib" target ="_blank" id= "profile-link">Shah's Github</a>
+        <li><a href="#contact">Contact</a></li>
+
+      </ul>
+      
+    </nav>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="welcome-section">
```

Comment:
It would be good to use semantic tags such as <section> for the welcome section to follow HTML/CSS best practices.